package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@planet/client",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
See https://github.com/planetlabs/planet-client-js#publishing-a-release

We tag and create releases from `master`.